sdcm/utils/raft/__init__.py

```diff
@@ -124,6 +128,15 @@ def __init__(self, node: "BaseNode") -> None:
     def is_enabled(self) -> bool:
         return True
 
+    @property
+    def consistent_topology_changes_enabled(self) -> bool:
```

Comment:
This function isn't correct now that this feature is enabled by default 

I would recommend reading the features of internal tables, and not from scylla.yaml

Reply:
@aleksbykov in his comment @fruch recommended to check if the feature is enabled by using internal tables instead of scylla_yaml.
Please check this option which would be safer.

Reply:
@aleksbykov 

this new logic basically mean we won't be able to run with SCT master with older releases of scylla
or worse we'll be able to run, but thing would fail unclearly, same the the issue we had in dtest cause of this kind of logic

I insist we should be reading the features out of scylla internal tables

Reply:
Fixed. Add new static methods and use them to check raft features state